code/ui/manager/src/components/sidebar/TreeNode.tsx

```diff
@@ -73,6 +73,7 @@ const BranchNode = styled.button<{
   background: 'transparent',
   '&:hover, &:focus': {
     background: theme.background.hoverable,
+    color: theme.color.lighter,
```

Comment:
Same here. Why did you choose to set it to use `color.lighter` ? These colors are set and would be the same if you are using dark and light mode.